src/fprime_gds/flask/events.py

```diff
@@ -21,8 +20,8 @@ class EventDictionary(DictionaryResource):
 
 class EventHistory(HistoryResourceBase):
     """
-    Resource supplying the history of events in the system. Includes `get_display_text` postprocessing to add in the
-    getter for the display text.
+    Resource supplying the history of events in the system. Includes postprocessing to add in the
+    for the display_text attribute which Flask will use.
     """
 
     def process(self, event):
```

Comment:
Side note, I was also considering to add the `display_text` attribute straight away at init-time as an attribute of the `ChData` / `EventData` classes, which could get rid of the need of this `process()` function. But I was less sure of the implications this would have on the Flask side and wanted to keep the PR tightly scoped.

Reply:
This was discussed and we're moving forward with it, see commits below.